src/Tribe/Aggregator/Event.php

```diff
@@ -146,12 +146,20 @@ public static function translate_service_data( $item ) {
 
 		if ( ! empty( $item->organizer ) ) {
 			$event['Organizer'] = array();
-			foreach ( $organizer_field_map as $origin_field => $target_field ) {
-				if ( ! isset( $item->organizer->$origin_field ) ) {
-					continue;
+			$organizer_entries  = $item->organizer;
```

Comment:
Can we rely on this always being an array (or should we add a check or cast before using in the foreach)? I'm sure this is covered, just asking to be safe.

Reply:
This is covered but a little more defensive programming will not hurt.